Reason: Fixes:
../../third_party/webrtc/modules/audio_processing/aec3/clockdrift_detector.h:34:3: error: unknown type name 'size_t'; did you mean 'std::size_t'?
  size_t stability_counter_;
  ^~~~~~
  std::size_t
/usr/bin/../lib/gcc/x86_64-alpine-linux-musl/10.2.0/../../../../include/c++/10.2.0/x86_64-alpine-linux-musl/bits/c++config.h:260:26: note: 'std::size_t' declared here
  typedef __SIZE_TYPE__         size_t;
                                ^
1 error generated.
Upstream: No

--- a/third_party/webrtc/modules/audio_processing/aec3/clockdrift_detector.h
+++ b/third_party/webrtc/modules/audio_processing/aec3/clockdrift_detector.h
@@ -12,6 +12,7 @@
 #define MODULES_AUDIO_PROCESSING_AEC3_CLOCKDRIFT_DETECTOR_H_
 
 #include <array>
+#include <stddef.h>
 
 namespace webrtc {
 
--- a/chrome/browser/search/background/ntp_backgrounds.h
+++ b/chrome/browser/search/background/ntp_backgrounds.h
@@ -6,6 +6,7 @@
 #define CHROME_BROWSER_SEARCH_BACKGROUND_NTP_BACKGROUNDS_H_
 
 #include <array>
+#include <stddef.h>
 
 class GURL;
 
